ci/format.bat

```diff
@@ -19,7 +19,7 @@ REM Test if Git is available on the Host
 where /q git || ECHO Error: Unable to find git in your PATH. && EXIT /B 1
 
 SET repo_dir=%SRC_DIR%\flutter
-SET ci_dir=%repo_dir%\flutter\ci
```

Comment:
I encountered many problems running the prepush hooks on Windows. I could not fix them all in time. But at least this one looks like a definite bug.